users.go

```diff
@@ -13,6 +13,7 @@ type User struct {
 	OfflineImageURL string `json:"offline_image_url"`
 	ViewCount       int    `json:"view_count"`
 	Email           string `json:"email"`
+	CreatedAt       string `json:"created_at"`
```

Comment:
This needs to be of type `Time`. See https://github.com/nicklaw5/helix/blob/main/time.go#L21-L37
```suggestion
	CreatedAt       Time `json:"created_at"`
```